NotificationService/NotificationService.BusinessLibrary/Providers/MSGraphNotificationProvider.cs

```diff
@@ -302,19 +317,11 @@ private async Task ProcessEntitiesInBatch(string applicationName, IList<EmailNot
                     // Mark these items as queued and Queue them
                     item.Status = NotificationItemStatus.Retrying;
                     item.ErrorMessage = itemResponse?.Error;
-                    if (itemResponse.Error?.Contains("quota was exceeded", StringComparison.InvariantCultureIgnoreCase) ?? false)
-                    {
-                        this.logger.WriteCustomEvent($"Mail Box Exhausted :  {item.EmailAccountUsed} ");
-                        this.logger.TraceInformation($"{itemResponse.Error} Item with notification id={item.NotificationId} will be retried with a different mail box", traceProps);
-                        if (!isAccountIndexIncremented)
-                        {
-                            isAccountIndexIncremented = true;
-                            this.emailAccountManager.IncrementIndex();
-                        }
-                    }
```

Comment:
why you deleted this?

Reply:
As i need to use the same code in case of meeting invite and email notification (individually), i moved this code to a method.